src/store/combineStore.js

```diff
@@ -1,10 +1,6 @@
-import thunk from 'redux-thunk';
-import { createStore, applyMiddleware } from 'redux';
-import { combineReducers } from '../reducers/mainReducer.js';
+import { createStore } from 'redux';
+import rootReducer from '../reducers/mainReducer.js';
```

Comment:
- Did you have to get rid of `redux-thunk`, why did you choose to remove it ?

Reply:
@kisakyegordon, since `thunk` is not being used at this stage, we (@myrdstom, @fahadmak and @Opio-Emmanuel-Omona  decided to remove it.
And also becasue it is mainly used while making asynchronous calls like fetch which is not being used at the moment.
However, if you have any insight on its necessity kindly let us know.

Reply:
- i will take your decision as a team, as we go along and  we require to make `asynchronous calls` you will add it.
-  for now i will merge without it